### PR TITLE
Q2.txt

### PR DESCRIPTION
**Pull request by NoorAlkattan**

### CODE REVIEW

File: HW10/read.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Java Script
+
+a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var i=a.length-3;
```
Comment (ssharba) [inline HW10/read.txt:4]: @NoorAlkattan it should start from the last element back to 0.

File: HW10/read.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	console.log (a[i].toUpperCase());
+	i=i+1;
+}
+
+
+Ruby
+
+a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"]
+
+ i=a.length-3
```
Comment (ssharba) [inline HW10/read.txt:16]: same here

File: HW10/read.txt (hunk 1 above)
Comment (NoorAlkattan) [reply HW10/read.txt:4]: you mean should be like this:
var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"]

 var i=a.length-1;

while ( i >= 0){
    console.log (a[i].toUpperCase());
    i=i-3;
 }

Comment (ssharba) [reply HW10/read.txt:4]: yes

**Pull request closed without merging**